controller/Makefile

```diff
@@ -50,6 +50,14 @@ deploy: build dev-release restart
 runserver:
 	python manage.py runserver
 
+postgres:
+	docker start postgres || docker run --restart="always" -d -p 5432:5432 --name postgres postgres:9.4.1
+	docker exec postgres createdb -U postgres deis 2>/dev/null || true
+	@echo "To use postgres for local development:"
+	@echo "    export export PGHOST=192.168.1.97"
```

Comment:
The word "export" is repeated here.

Instead of hard-coding a particular local IP, this should call `boot2docker` like [`make dev-registry`](https://github.com/deis/deis/blob/master/Makefile#L23)--then at least on Macs we get the right IP. Or else use `[container IP address]`.